zkevm-circuits/src/evm_circuit/execution/create.rs

```diff
@@ -62,6 +62,8 @@ pub(crate) struct CreateGadget<F> {
 
     code_hash: Cell<F>,
 
+    code_hash_previous: Cell<F>,
+    is_code_hash_previous_zero: IsZeroGadget<F>,
```

Comment:
Just a subjective opinion from my side:

I feel we could rename this to `not_collision`, so that its easier to interpret the condition guards that follow in the circuit.

But entirely up to you, this is what I would have preferred.

Reply:
good name , I like it !

Reply:
change name to `not_address_collision` in [5ad0c3d](https://github.com/scroll-tech/zkevm-circuits/pull/406/commits/5ad0c3d723646200ac2958cb1d362b17c40156c0) thanks!